pkg/daemon/reader.go

```diff
@@ -40,6 +40,10 @@ func pollDevice(
 		return activeCard, removed, err
 	}
 
+	if count > 1 {
+		log.Info().Msg("More than one card on the reader")
+	}
```

Comment:
this is here for debugging purpose only, count = 2 isn't stable. No changes made for now, maybe material for next piece of work